src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -267,10 +293,13 @@ class RelayToTIRVisitor : public MixedModeMutator {
     // %3 = qnn.requantize(%2, %req_input_scale_scalar, %req_input_zero_point_scalar,
     //                     %output_scale_scalar, %output_zero_point_scalar)
     // clip(%3, a_min=%min_scalar, a_max=%max_scalar)
-    tir::Var input("input", DataType::Handle(8));
-    tir::Var filter("filter", DataType::Handle(8));
-    tir::Var bias("bias", DataType::Handle(32));
-    tir::Var output("output", DataType::Handle(8));
+    BufferCreator buffer_creator_;
```

Comment:
I have seen trailing underscores for member variables. Not sure what the guideline is.

Reply:
You are right! ref : https://google.github.io/styleguide/cppguide.html#Local_Variables